gtp2ogs.js

```diff
@@ -418,34 +418,84 @@ process.on('uncaughtException', function (er) {
+}
+
+if (argv.minmaintime && (argv.minmaintimeranked || argv.minmaintimeunranked)) {
+    console.log("Warning: You are using --minmaintime in combination with --minmaintimeranked or/and --minmaintimeunranked. \n Use either --minmaintime alone , OR --minmaintimeranked with --minmaintimeunranked. \n But don't use the 3 --minmaintime arguments at the same time. " );
+}
+
+if (argv.maxperiods && (argv.maxperiodsranked || argv.maxperiodsunranked)) {
+    console.log("Warning: You are using --maxperiods in combination with --maxperiodsranked or/and --maxperiodsunranked. \n Use either --maxperiods alone , OR --maxperiodsranked with --maxperiodsunranked. \n But don't use the 3 --maxperiods arguments at the same time. " );
+}
+
+if (argv.minperiods && (argv.minperiodsranked || argv.minperiodsunranked)) {
+    console.log("Warning: You are using --minperiods in combination with --minperiodsranked or/and --minperiodsunranked. \n Use either --minperiods alone , OR --minperiodsranked with --minperiodsunranked. \n But don't use the 3 --minperiods arguments at the same time. " );
+}
+
+if (argv.maxperiodtime && (argv.maxperiodtimeranked || argv.maxperiodtimeunranked)) {
+    console.log("Warning: You are using --maxperiodtime in combination with --maxperiodtimeranked or/and --maxperiodtimeunranked. \n Use either --maxperiodtime alone , OR --maxperiodtimeranked with --maxperiodtimeunranked. \n But don't use the 3 --maxperiodtime arguments at the same time. " );
+}
+
+if (argv.minperiodtime && (argv.minperiodtimeranked || argv.minperiodtimeunranked)) {
+    console.log("Warning: You are using --minperiodtime in combination with --minperiodtimeranked or/and --minperiodtimeunranked. \n Use either --minperiodtime alone , OR --minperiodtimeranked with --minperiodtimeunranked. \n But don't use the 3 --minperiodtime arguments at the same time. " );
+}
+
+if (argv.noautohandicap && (argv.noautohandicapranked || argv.noautohandicapunranked)) {
+    console.log("Warning: You are using --noautohandicap in combination with --noautohandicapranked or/and --noautohandicapunranked. \n Use either --noautohandicap alone , OR --noautohandicapranked with --noautohandicapunranked. \n But don't use the 3 --noautohandicap arguments at the same time. " );
+}
+
+if (argv.nopause && (argv.nopauseranked || argv.nopauseunranked)) {
+    console.log("Warning: You are using --nopause in combination with --nopauseranked or/and --nopauseunranked. \n Use either --nopause alone , OR --nopauseranked with --nopauseunranked. \n But don't use the 3 --nopause arguments at the same time. " );
+}
+
+// console warnings : depreciated features if used
+
 if (argv.botid) {
-    console.log("Warning: --botid alias is no longer supported. Use --username instead.");
+    console.log("Warning: --botid alias is no longer supported. Use --username instead." );
```

Comment:
Stop adding random spaces at the end of the line. `" );` => `" );`. This space was not there in the original code.

Reply:
done :1st_place_medal: